app/forms/waste_exemptions_engine/check_registered_name_and_address_form.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+module WasteExemptionsEngine
+  class CheckRegisteredNameAndAddressForm < BaseForm
+    delegate :temp_use_registered_company_details, to: :transient_registration
+    delegate :company_no, to: :transient_registration
+    delegate :registered_company_name, to: :transient_registration
+    delegate :operator_name, to: :transient_registration
```

Comment:
There is no operator_name attribute on transient_registration. Presumably this isn't used anyway.

Reply:
As discussed - this is fine as it is, my understanding of this pattern needs work.